pandas/core/series.py

```diff
@@ -4684,7 +4684,8 @@ def between(self, left, right, inclusive=True) -> Series:
             Left boundary.
         right : scalar or list-like
             Right boundary.
-        inclusive : bool, default True
+        inclusive : boolean or str, default "True"
```

Comment:
As before do the docs become: `inclusive : {"both", "neither", "left", "right"}`

where True and False are silently accepted.

Reply:
How do you suggest that we write this line of documentation if inclusive has a standard value of a string ("both"), but also allows boolean values for backward compatibility?

Reply:
I would probably do something like:

```
include : {"both", "neither", "left", "right"}
    Whether to set each bound as closed or open. For backwards compatibility this can also 
    be set to ``True`` ("both"), False ("neither") or None (..) 
```

You might find this is a bit more controversial with the core devs but I think it makes it quite evident that the users is supposed to adopt the recommended strings.